src/Imgix/UrlBuilder.php

```diff
@@ -27,31 +25,27 @@ class UrlBuilder {
     const MAX_WIDTH = 8192;
     const SRCSET_WIDTH_TOLERANCE = 0.08;
 
-    // constants cannot be dynamically assigned; keeping as a class variable instead
-    private $targetWidths;
```

Comment:
Same for this one as well.

Reply:
This class property is [defined here](https://github.com/imgix/imgix-php/blob/ce9b921f665444265440ec9288701a32a0103d2d/src/Imgix/UrlBuilder.php#L30-L31), and [assigned a value in the constructor](https://github.com/imgix/imgix-php/blob/ce9b921f665444265440ec9288701a32a0103d2d/src/Imgix/UrlBuilder.php#L45), but then never used later on in the file (by calling `$this->targetWidths`).

There is [a method defined with the same name](https://github.com/imgix/imgix-php/blob/ce9b921f665444265440ec9288701a32a0103d2d/src/Imgix/UrlBuilder.php#L127-L130) which is [used at one point](https://github.com/imgix/imgix-php/blob/ce9b921f665444265440ec9288701a32a0103d2d/src/Imgix/UrlBuilder.php#L104), but it's assigned to a local variable.

And since it is a `private` property, no one is using this in their business code directly.

Let me know if you need more info! 👍

Reply:
This is one of those times I wished I hadn't checked the git blame 🙃 

![](https://i.imgflip.com/4/3llic2.jpg)

Reply:
But thanks for the thorough explanation, that makes sense to me.